src/main/java/at/gammastrahlung/monopoly_server/network/websocket/MonopolyMessageHandler.java

```diff
@@ -241,11 +255,7 @@ private static ServerMessage rollDice(ClientMessage clientMessage, WebSocketPlay
 
         game.rollDice();
 
-        return ServerMessage.builder()
-                .messagePath("roll_dice")
-                .type(ServerMessage.MessageType.INFO)
-                .game(game)
-                .build();
+        return ServerMessage.builder().messagePath("roll_dice").type(ServerMessage.MessageType.INFO).game(game).build();
```

Comment:
please revert this, it's hard to read and not an improvement, makes retention worse for no reason